app/controllers/questions_controller.rb

```diff
@@ -29,9 +29,17 @@ def show
     @question = @questions[(params[:id]).to_i - 1]["text"]
     @answers = @questions[(params[:id]).to_i - 1]["answers"]
     @answer = Answer.new
+  end
 
+  def next
+    @questions = get_question_list()
+    @next_question_id = next_question(@questions, [params[:id]])
+  end 
 
-  end
+   def previous
+    @questions = get_question_list()
+    @prev_question_id = @question.prev_question[params[:id]]
+  end 
```

Comment:
This bundle of work doesn't include the `next` and `previous` functionality, so can you remove these changes before we commit it to the main project?